testsupport/init.go

```diff
@@ -86,9 +87,9 @@ func waitForOperators(t *testing.T) {
 	initHostAwait.RegistrationServiceURL = registrationServiceURL
 
 	// wait for member operators to be ready
-	initMemberAwait = getMemberAwaitility(t, cl, initHostAwait, memberNs)
+	initMemberAwait = getMemberAwaitility(t, cl, initHostAwait, kubeconfig, memberNs)
 
-	initMember2Await = getMemberAwaitility(t, cl, initHostAwait, memberNs2)
+	initMember2Await = getMemberAwaitility(t, cl, initHostAwait, kubeconfig, memberNs2)
 
 	hostToolchainCluster, err := initMemberAwait.WaitForToolchainClusterWithCondition(t, "e2e", hostNs, wait.ReadyToolchainCluster)
 	require.NoError(t, err)
```

Comment:
don't forget to drop these and the following lines too

Reply:
yea updated it now